### PR TITLE
Release notes for 3.7.0

### PR DESCRIPTION
**Pull request by scragraham**
Draft of release notes for 3.7.0

### CODE REVIEW

File: RELEASE [hunk 1]
```diff
@@ -5,6 +5,94 @@
...
+ - Terminate not started AppServers first
+ - Check for valid local source archive
+ - Project ID validation upgrade
+ - Application scaling for version configuration
+ - Added service ID validation and modified version ID validation.
+ - Implement apps.services.list
+ - Exit early if version is not found in zookeeper
+ - Use zookeeper for stopped versions
+ - Ec2 private ips
+ - Defer to vendored framework for endpoints
```
Comment (sjones4) [inline RELEASE]:
```suggestion
 - Defer to vendored framework for endpoints 2
```

File: RELEASE [hunk 2]
```diff
@@ -5,6 +5,94 @@
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
       /_/   /_/
 
+AppScale version 3.7.0, released March 2019
```
Comment (obino) [inline RELEASE]: April I guess ...

File: RELEASE [hunk 3]
```diff
@@ -5,6 +5,94 @@
...
+Highlights of features and defects fixed in this release:
+ - IaaS restructure
```
Comment (obino) [inline RELEASE]: InfrastructureManager

File: RELEASE (hunk 2 above)
Comment (scragraham) [inline RELEASE]:
```suggestion
AppScale version 3.7.0, released April 2019
```

File: RELEASE [hunk 4]
```diff
@@ -5,6 +5,94 @@
 /_/  |_/ .___/ .___//____/\___/\__,_/_/ \___/
       /_/   /_/
 
+AppScale version 3.7.0, released April 2019
+Highlights of features and defects fixed in this release:
+ - IaaS restructure
```
Comment (scragraham) [inline RELEASE]:
```suggestion
 - InfrastructureManager restructure
```

**Pull request merged**